README.md

```diff
@@ -69,76 +66,68 @@ The IBM Blockchain Platform extension provides an explorer and commands accessib
 
 <img src="/client/media/start_new_project_small.gif" width="85%" alt="Start a new Fabric smart contract project">
-<!---Link to docs with further instructions
---->
+
 
 ### Edit / write smart contract files
-<!---Short explanation with code-blocks
---->
+You'll find the smart contract file in the `lib` directory of a newly generated project.
+
+<img src="/client/media/edit_write_files.png" width="85%" alt="Edit smart contract files">
 
-<img src="/client/media/edit_write_files.png" width="85%" alt="Edit smart contract files" width="85%" alt="Edit smart contract files">
-<!---Link to docs with further instructions
---->
 
 ### Package a smart contract project
+To package a project you have open in your workspace, run the `Package a Smart Contract Package` command. Packages will be added to the `Smart Contract Packages` panel in the explorer view.
+
+Alternatively, navigate to the explorer view (click the IBM Blockchain Platform icon in the left-navigation) then click the `add` icon (this will again package up the project you have open in your workspace).
 
 <img src="/client/media/package_project.png" width="85%" alt="Create a package from a smart contract project">
 
+### Export a smart contract package
+Right-click a package and select the `Export Package` option.  You can select where the package is exported to. Use this to take the packages you have made and deploy them somewhere else.
 
-<!---Short explanation with code-blocks
---->
-<!---Screenshot of UI/Video of prototype click-through
---->
-<!---Link to docs with further instructions
---->
+<img src="/client/media/export_smart_contract_package.png" width="85%" alt="Export a smart contract project">
 
-<!---Things you can do in the Fabric view once the extension is installed
---->
-<!---Introduction to Fabric view
---->
 ### Connect to local_fabric runtime
+The `local_fabric` runtime can be enabled by left-clicking it in the `Blockchain Connections` panel. The first time you enable it, Fabric images will be installed.  
+Left-click to connect to `local_fabric` when it is enabled. Right-click the connection for management options (including start/stop, teardown and toggle dev mode).
 
 <img src="/client/media/connect_to_local_fabric_runtime_small.gif" width="85%" alt="Connect to local_fabric runtime">
 
-<!---Short explanation with code-blocks
---->
-<!---Screenshot of UI/Video of prototype click-through
---->
-<!---Link to docs with further instructions
---->
+### Teardown the local_fabric runtime
+When you start/stop the local_fabric, some data will be maintained. To completely teardown the runtime and start over, right-click `local_fabric` in the `Blockchain Connections` view and choose `Teardown Fabric Runtime` (you must then confirm this action).
```

Comment:
think you need to change "some data" to be the world state will be maintained or all data will be maintained